announcements/2021-Week-03/announcement.md

```diff
@@ -0,0 +1,26 @@
+# :zap: This Week in Provider-Independent Announcements (January 22, 2021):zap:
+
+Please note that we are aligning some key terms in our product communication with the Kubernetes project's terminology:
+
+- The term *management cluster* replaces the term *control plane*. The Kubernetes project uses the term control plane when dealing with the master nodes (as they were called formerly) of a cluster, so we are removing an ambiguity that confused people.
+- The API formerly called *Control Plane Kubernetes API* is now called *Management API*.
+- What we called *tenant cluster* is now called a *workload cluster*.
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+[Prometheus Operator v0.5.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.5.0/) upgrades to the chart with Prometheus v2.22.1.
+
+## Web UI
+
+## kubectl gs
+
+- Added support for getting nodepools and removed mention of the unexistent ‘create cluster’ command.
```

Comment:
@whites11 is this for Azure only or AWS also?

Reply:
Both Azure and AWS

Reply:
Thanks!